.github/workflows/ci.yml

```diff
@@ -23,3 +23,25 @@ jobs:
         npm test
       env:
         CI: true
+
+  lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build, and lint
+      run: |
+        npm ci
+        npm start
+        npm run lint
+      env:
+        CI: true
+        GAPI_MAX_PARALLEL: 3
```

Comment:
I did some rough tests to come up with this number, i.e. see:

* https://github.com/namoscato/google-api-typings-generator/runs/374746191
* https://github.com/namoscato/google-api-typings-generator/runs/374752209